doc/source/train/api/api.rst

```diff
@@ -150,6 +84,34 @@ PyTorch Lightning
     `LightningCheckpoint`, and `LightningPredictor` in Ray 2.8. Please 
     refer to the :ref:`migration guides <migration-guide>` for more info.
 
+HuggingFace Transformers
+************************
+
+.. note::
+
+    We will deprecate `TransformersTrainer`, `TransformersCheckpoint`
+    in Ray 2.8. Please refer to the :ref:`migration guides <migration-guide>` 
+    for more info.
```

Comment:
Hm let's leave this out until we actually have the migration guides?

Reply:
ok